Assets/MRTK/Core/Interfaces/InputSystem/IMixedRealityNearPointer.cs

```diff
@@ -28,10 +28,22 @@ public interface IMixedRealityNearPointer : IMixedRealityPointer
         ///
         /// NOTE: Not all near pointers have a grasp point (for example a poke pointer).
         /// </summary>
-        /// <param name="rotation">Out parameter filled with the grasp position if available, otherwise <see href="https://docs.unity3d.com/ScriptReference/Vector3-zero.html">Vector3.zero</see>.</param>
+        /// <param name="position">Out parameter filled with the grasp position if available, otherwise <see href="https://docs.unity3d.com/ScriptReference/Vector3-zero.html">Vector3.zero</see>.</param>
         /// <returns>True if a grasp point was retrieved, false if not.</returns>
         bool TryGetNearGraspPoint(out Vector3 position);
 
+        /// <summary>
+        /// Because pointers shouldn't be able to interact with objects that are "behind" it, it is necessary to determine the forward axis of the pointer when making interaction checks.
+        /// 
+        /// For example, a grab pointer's axis should is the result of Vector3.Lerp(palm forward axis, palm to index finger axis).
+        ///
+        /// This method provides a mechanism to get this forward axis. It should be normalized.
+        /// </summary>
+        /// <param name="axis">Out parameter filled with the grasp's forward axis if available, otherwise returns the forward axis of the transform.</param>
+        /// <returns>True if a grasp's forward axis was retrieved, false if not.</returns>
+        bool TryGetNearGraspAxis(out Vector3 axis);
```

Comment:
Do you think this needs to be added to the interface? It's possible I missed a reference, but as far as I can tell, this is only used internally to `SpherePointer`.

Reply:
It's only used in sphere pointer right now, but I believe it is the same general category of function as TryGetNearGraspPoint, so it can be useful if we need to implement a visual direction indicator or something similar

Reply:
Adding this to the interface makes this PR a [breaking change](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/BreakingChanges.html) for existing implementers of the interface, which would be best to avoid if possible. It's also not necessarily relevant to all near pointers, which includes the poke pointer which can't "grasp" (i feel the same way about `TryGetNearGraspPoint`). I think to prevent the breaking change, and to prevent needing to add this to the poke pointer where the concept isn't possible, it might be best to hold off adding it to the interface.